api/util/xray.py

```diff
@@ -46,13 +47,22 @@ def setup_xray(cls):
         httplib_add_ignored(hostname="logs.*.amazonaws.com")
 
     @classmethod
-    def configure_app(cls, app: Flask):
+    def enabled(cls) -> bool:
         enable_xray = os.environ.get(cls.XRAY_ENV_ENABLE)
-        if enable_xray and enable_xray.lower() == "true":
+        return enable_xray and enable_xray.lower() == "true"
+
+    @classmethod
+    def configure_app(cls, app: Flask):
+        if cls.enabled():
             logging.getLogger().info("Configuring app with AWS XRAY.")
             cls.setup_xray()
             PalaceXrayMiddleware(app, xray_recorder)
 
+    @classmethod
+    def include_barcode(cls):
+        include_barcode = os.environ.get(cls.XRAY_ENV_ENABLE, "true")
```

Comment:
Did you mean `XRAY_ENV_PATRON_BARCODE` here?

Reply:
I sure did. Good catch, thank you. Just pushed a fix for it.